dashboard/app/helpers/users_helper.rb

```diff
@@ -9,6 +9,17 @@ def summarize_user_progress(script, user = current_user, exclude_level_progress
     user_data = {}
     merge_user_summary(user_data, user)
     merge_script_progress(user_data, user, script, exclude_level_progress)
+
+    peer_review_summaries = PeerReview.get_peer_review_summaries(user, script)
+
+    unless peer_review_summaries.empty?
+      peer_review_summaries.each do |summary|
+        summary.merge!(url: peer_review_path(summary[:id]))
+      end
+
+      user_data[:peerReviewsPerformed] = peer_review_summaries
+    end
```

Comment:
Simplify to this?
```ruby
user_data[:peerReviewsPerformed] = PeerReview.get_peer_review_summaries(user, script).map do |summary|
  summary.merge(url: peer_review_path(summary[:id]))
end
```
(Can return `user_data.compact` if you're worried about the possible nil value.)